ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/objectstore/CachedTest.java

```diff
@@ -71,11 +71,17 @@ public void run()
                 Thread.yield();
                 
                 if (s != null) {
-                    if (participantStore.remove_committed(u, TYPE))
+                    if (participantStore.remove_committed(u, TYPE)) {
                         passed = true;
+                    } else {
+                        System.err.println("Could not remove "+u);
```

Comment:
I think logger should be used instead of the `System.err.println`

Reply:
I would agree if it were a statement I was introducing.

That statement was already in the code and not actually added by this patch. I will look to add a second commit against https://issues.jboss.org/browse/JBTM-2880

Reply:
I just realised, this is in the test sorry. I think it is fine to keep it as it is

Reply:
right, I missed. sorry.